### PR TITLE
Add heading2

### PR DESCRIPTION
**Pull request by lumina293**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -10,3 +10,6 @@
   color: green;
   font-weight: bold;
 }
+#heading2{
```
Comment (briannqc) [inline styles.css:14]: Can we add an empty line between the selector?

File: styles.css [hunk 2]
```diff
@@ -10,3 +10,6 @@
...
+  color: blueviolet;
+}
```
Comment (briannqc) [inline styles.css]: Can we add a tailing empty line?

File: styles.css (hunk 1 above)
Comment (lumina293) [reply styles.css:14]: Done

File: styles.css (hunk 2 above)
Comment (lumina293) [reply styles.css]: Updated

**Review by briannqc — APPROVED** [review]
LGTM

**Pull request merged**